pkg/reconciler/instances/loader.go

```diff
@@ -27,4 +29,6 @@ import (
 	_ "github.com/kyma-incubator/reconciler/pkg/reconciler/instances/scmigration"
 	//import required to register component reconciler 'serverless' in reconciler registry
 	_ "github.com/kyma-incubator/reconciler/pkg/reconciler/instances/serverless"
+	//import required to register component reconciler 'utils' in reconciler registry
+	_ "github.com/kyma-incubator/reconciler/pkg/reconciler/instances/utils"
```

Comment:
This package is wrongly located and has to be remove urgently (see #549 )

Reply:
it's created by mechanism to create new component reconcilers. It could be removed when mentioned pr will be merged